vivisect/qt/funcgraph.py

```diff
@@ -83,16 +84,29 @@ def contextMenuEvent(self, event):
         else:
             menu = QtGui.QMenu(parent=self)
 
-        viewmenu = menu.addMenu('view   ')
-        viewmenu.addAction("Save frame to HTML", ACT(self._menuSaveToHtml))
-        #TODO: Refresh menu item :)
+        self.viewmenu = menu.addMenu('view   ')
+        self.viewmenu.addAction("Save frame to HTML", ACT(self._menuSaveToHtml))
+        self.viewmenu.addAction("Refresh", ACT(self.refresh))
 
         menu.exec_(event.globalPos())
 
+    def setRefreshCallback(self, callback):
+        '''
+        Set a refresh "callback" that will be called whenever a user input
+        calls for refreshing the graph.
+        '''
+        self._refresh_cb = callback
+
```

Comment:
this seems like a likely use case for a proper Qt signal/slot rather than an accessor for a one-off variable ( would also allow cleaner glueing to ui triggered events and more than one "callback" )